projects/WTI-API/src/main/config/WebServer.java

```diff
@@ -2,6 +2,7 @@
 
 import java.net.URISyntaxException;
 import java.util.EnumSet;
+import java.util.logging.ConsoleHandler;
```

Comment:
This added import does not appear to be used anywhere, which will generate at least a warning, maybe an error, in Eclipse.  If there's no need for the import, please remove it.

Reply:
removed, pushed fix.